operator/gitops/argocd/pipeline-service/tekton-results/base/kustomization.yaml

```diff
@@ -26,3 +24,80 @@ patchesJson6902:
         path: "/metadata/labels"
         value:
           argocd.argoproj.io/managed-by: openshift-gitops
+  - target:
+      kind: Service
+      name: tekton-results-api-service
+      namespace: tekton-results
+    patch: |-
+      - op: add
+        path: "/metadata/annotations"
+        value:
+          service.beta.openshift.io/serving-cert-secret-name: tekton-results-tls
+  # the secret provided by stonesoup have data fields in the form db.*, patch the env passed to the api to match the fields
+  # remove default values for database env since all the values are provided by the secret.
+  - target:
+      kind: Deployment
+      name: tekton-results-api
+      namespace: tekton-results
+    patch: |-
+      - op: remove
+        path: /spec/template/spec/containers/0/env/3/value
+      - op: add
+        path: /spec/template/spec/containers/0/env/3/valueFrom
+        value:
+          secretKeyRef:
+            key: db.name
+            name: tekton-results-database
+      - op: remove
+        path: /spec/template/spec/containers/0/env/2/value
+      - op: add
+        path: /spec/template/spec/containers/0/env/2/valueFrom
+        value:
+          secretKeyRef:
+            key: db.host
+            name: tekton-results-database
+      - op: replace
+        path: /spec/template/spec/containers/0/env/1/valueFrom/secretKeyRef/key
+        value: db.password
+      - op: replace
+        path: /spec/template/spec/containers/0/env/1/valueFrom/secretKeyRef/name
+        value: tekton-results-database
+      - op: replace
+        path: /spec/template/spec/containers/0/env/0/valueFrom/secretKeyRef/key
+        value: db.user
+      - op: replace
+        path: /spec/template/spec/containers/0/env/0/valueFrom/secretKeyRef/name
+        value: tekton-results-database
+  # we are using tekton-results namespace, so patch the arguments to watcher
+  - target:
+      kind: Deployment
+      name: tekton-results-watcher
+      namespace: tekton-results
+    patch: |-
+      - op: replace
+        path: /spec/template/spec/containers/0/args/1
+        value: tekton-results-api-service.tekton-results.svc.cluster.local:50051
+
+patchesStrategicMerge:
+  # Remove local database deployment for staging, for local testing manifests are in developer/local/postgres
+  - |-
```

Comment:
Am I right in understanding that by default tekton-results will deploy a postgres DB?

Reply:
Yes, Adam has added a PR to change this behaviour upstream. Once that merges I will update the manifests here.